cs-powershell/src/main/java/io/cloudslang/content/entities/WSManRequestInputs.java

```diff
@@ -28,6 +28,7 @@
     private String trustKeystore;
     private String trustPassword;
     private String script;
+    private String modules;
```

Comment:
because all the attributes are initialized in the constructor, you can make them all final

Reply:
Is this your final fantasy? I will grant you this one.